Cabal/Distribution/Backpack/ConfiguredComponent.hs

```diff
@@ -267,3 +269,12 @@ newPackageDepsBehaviourMinVersion = mkVersion [1,7,1]
 newPackageDepsBehaviour :: PackageDescription -> Bool
 newPackageDepsBehaviour pkg =
    specVersion pkg >= newPackageDepsBehaviourMinVersion
+
+fixFakePkgName :: PackageDescription -> PackageName -> (PackageName, ComponentName)
```

Comment:
I had to read the call sites to figure out what this was doing. I guess it's going away soon though...